### DISCUSSION TITLE
Job that runs every 2 hours, but spawns new jobs? Use node-cron or Bullmq?

### DISCUSSION THREAD

**Discussion by notflip** [taskforcesh/bullmq#1995] [Q&A]
I have a situation in which, every 2 hours, I need to fetch records in the database, and for each record create a job.

Now I saw that BullMQ has repeatable jobs, is it considered a good idea to use BullMQ repeatable job to spawn other jobs? 
Or is it easier to just use node-cron to create the necessary jobs every 2 hours?

**Comment by manast** [comment]
I use that pattern in my places and it works nicely.

**Reply by manast** [reply]
Btw, I think the best is to keep one queue dedicated only for the repeatable job that will spawn new jobs, that gets easy to debug and maintain later on.

**Reply by notflip** [reply]
Thanks @manast ! I'm going to give it a shot. Is it a public repo, in which you're doing this? Would love to have a look if it is!
